src/pkg/Directory.Build.targets

```diff
@@ -10,7 +10,11 @@
   -->
   <Target Name="GetInstallerBrandingNames"
           DependsOnTargets="GetProductVersions">
-    <PropertyGroup Condition="'$(PreReleaseVersionLabel)' != '' and $(PreReleaseVersionLabel.StartsWith('preview'))">
+    <PropertyGroup
```

Comment:
Why have this special case for preview rather than updating both PreReleaseVersionLabel and ReleaseBrandSuffix for preview release in eng/Versions.props?

Reply:
This logic let me remove the `previewX` `Preview X` duplication in `eng/Versions.props`, and I considered making it also work for `alphaX` `Preview X`, but for expedience I'm going to let the `alpha1` vs. `Alpha 1` duplication live on for now.

Ultimately, I want to make this handle `[a-z]+[0-9]+` to get rid of the duplication and allow anything.